telemetry-testing/src/test/java/io/opentelemetry/example/telemetry/ApplicationTest.java

```diff
@@ -80,7 +83,7 @@ public void testTelemetry() {
               var spans = extractSpansFromRequests(requests);
               assertThat(spans)
                   .extracting(Span::getName)
-                  .contains("Controller.doWork", "Controller.ping");
+                  .contains("Controller.doWork", "GET", "GET /ping");
```

Comment:
I'm surprised this passed, I thought we disabled `Controller.doWork` spans by default

Reply:
Yeah, it's because [the method is explicitly annotated](https://github.com/open-telemetry/opentelemetry-java-examples/blob/main/telemetry-testing/src/main/java/io/opentelemetry/example/telemetry/Controller.java#L41) with `@WithSpan`.